source/seedsource/tasks/write_tif.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import rasterio
+from ncdjango.geoprocessing.data import Raster
+from ncdjango.geoprocessing.params import RasterParameter, StringParameter, IntParameter
+from ncdjango.geoprocessing.workflow import Task
+
+from seedsource_project import settings
+import os.path
+import tempfile
+
+
+class WriteTif(Task):
+    name = 'write_tif'
+    inputs = [
+        RasterParameter('variable'),
+    ]
+    outputs = [StringParameter('filename')]
+
+    def execute(self, variable):
+        fd, filename = tempfile.mkstemp(prefix=settings.NC_TEMPORARY_FILE_LOCATION, suffix='.tif')
```

Comment:
I don't think we want to reuse `NC_TEMPORARY_FILE_LOCATION` for this, as we need to serve static file from the chosen directly, and we *don't* want to serve the temp files directory. How about a new setting for this purpose, like `DATASET_DOWNLOAD_DIR`?